pkg/sql/logictest/logic.go

```diff
@@ -1658,6 +1664,32 @@ func (t *logicTest) processSubtest(
 			}
 
 			if !s.skip {
+				if query.kvtrace {
+					_, err := t.db.Exec("SET TRACING=on,kv")
+					if err != nil {
+						return err
+					}
+					_, err = t.db.Exec(query.sql)
+					if err != nil {
+						t.Error(err)
+					}
+					_, err = t.db.Exec("SET TRACING=off")
+					if err != nil {
+						return err
+					}
+					query.colTypes = "T"
+					query.sql = `SELECT message FROM [SHOW KV TRACE FOR SESSION] WITH ORDINALITY
```

Comment:
Why do you need `WITH ORDINALITY` here?

Reply:
I don't know, I just cargo culted it from somewhere else :) I'll get rid of it.